### PR TITLE
Assert rater_functions

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

File: R/rater_functions.R [hunk 1]
```diff
@@ -6,6 +6,11 @@
 ### randomize.order: soll die Reihenfolge der Pseudocodes nach Zufall bestimmt werden?
 make.pseudo <- function(datLong, idCol, varCol, codCol, valueCol, n.pseudo, randomize.order = TRUE, verbose = FALSE)   {
       datLong     <- eatTools::makeDataFrame(datLong)
+      lapply(c(idCol, varCol, codCol), checkmate::assert_scalar)
+      checkmate::assert_vector(valueCol)
```
Comment (grewered) [inline R/rater_functions.R:10]: what kind of values does valueCol have? numeric or chr or both?

File: R/rater_functions.R [hunk 2]
```diff
@@ -61,7 +69,10 @@ meanKappa <- function( dat , type = c("Cohen", "BrennanPrediger"), weight = "unw
   # weight.mean ... = T, if agreement is weighted by number of rater subjects,
   #            = F, if it is averaged among all rater pairs
   dat  <- eatTools::makeDataFrame(dat)
+  checkmate::assert_character(weight, len = 1)
+  checkmate::assert_logical(weight.mean, len = 1)
   type <- match.arg(type)
+
```
Comment (grewered) [inline R/rater_functions.R:75]: I understood line 74 as: it gives `type` a DEFAULT value (Cohen).
should check type for chr and len = 1; or as a subset of c(Cohen, BrennanPrediger)? or just leave it as it is?

Comment (sachseka) [reply R/rater_functions.R:75]: yes, please as a subset of c("Cohen", "BrennanPrediger")

File: R/rater_functions.R (hunk 1 above)
Comment (sachseka) [reply R/rater_functions.R:10]: this should be just another column identifier just as idcol, varcol etc. see man page

**Review by sachseka — CHANGES_REQUESTED** [review]

**Comment by sachseka** [comment]
Please pay attention to checks

File: R/rater_functions.R [hunk 3]
```diff
@@ -61,7 +68,11 @@ meanKappa <- function( dat , type = c("Cohen", "BrennanPrediger"), weight = "unw
   # weight.mean ... = T, if agreement is weighted by number of rater subjects,
   #            = F, if it is averaged among all rater pairs
   dat  <- eatTools::makeDataFrame(dat)
+  checkmate::assert_character(weight, len = 1)
+  checkmate::assert_logical(weight.mean, len = 1)
   type <- match.arg(type)
```
Comment (grewered) [inline R/rater_functions.R:73]: ok, so when I used assert_choice (similar to subset, but allows only scalar input) it doesn't work, but with the `match.arg` line it does, so I'll leave it like that?

Comment (sachseka) [reply R/rater_functions.R]: Yes please leave match.arg

**Review by sachseka — APPROVED** [review]
match.arg is fine

**Pull request merged**